Apps/FCM/ios/Podfile

```diff
@@ -49,17 +39,22 @@ target 'FCMSampleApp' do
     :app_path => "#{Pod::Config.instance.installation_root}/.."
   )
 
+  pod 'customerio-reactnative/fcm', :path => '../node_modules/customerio-reactnative'
```

Comment:
Was there a reason this was moved? Will customers need to move it in their `Podfile`s?

Reply:
The change in position does not affect the functioning. The customers will not have to move anything in their podfile..